### ISSUE TITLE
Feature request, set "required" attribute on input

### ISSUE THREAD

**Issue by reviewingcode** [mattzollinhofer/vue-typeahead-bootstrap#76]
Hi,

I stumbled on this fork while checking out the original by the name `vue-bootstrap-typeahead` and this is very nice work. Thanks to all maintainers for improving and keeping this alive.

This component is 99% percent what I needed. I couldn't find a way to set `required` attribute for input. I need that to trigger native form validation on `submit` event.

Currently I'm doing it with this code block placed in `updated` lifecycle hook which works but feels hacky and kinda bloats the code.
```
const ref = this.$refs['autocomplete']
    
    if(ref) {
      ref.$el.querySelector('.autocomplete-input').setAttribute('required', true)
    }
```

I'd really appreciate if there is a custom prop to set `required` attribute on the input tag.

**Comment by mattzollinhofer** [comment]
Yeah, this seems totally reasonable to me. I'd be happy for this feature to be implemented.

I think we could do this eventually, but it might take a bit. If you (or anyone) is interested in sending a PR over, I think it might be fairly attainable.
